packages/cli/src/commands/deploy/aws-providers/serverless.js

```diff
@@ -7,28 +9,25 @@ export const preRequisites = [
       'Please follow the steps at https://www.serverless.com/framework/docs/providers/aws/guide/installation/ to install Serverless.',
```

Comment:
This link seems to be broken now

Reply:
The link does work for me. However I'd recommend this one to be safe:

```suggestion
      'Please follow the steps at https://www.serverless.com/framework/docs/getting-started to install Serverless.',
```